python_modules/dagster/dagster/_core/definitions/freshness_policy.py

```diff
@@ -34,12 +34,29 @@ class FreshnessPolicy(
     cron_schedule is set), the data used to produce the current version of this asset must be no
     older than `maximum_lag_minutes` before the most recent available data.
 
-    Examples:
+    The freshness status of assets with policies defined will be visible in the UI. If you are using
+    an asset reconciliation sensor, this sensor will kick off runs to help keep your assets up to
+    date with respect to their FreshnessPolicy.
+
+    Args:
+        maximum_lag_minutes (float): An upper bound for how old the data contained within this
+            asset may be, relative to the current time.
+        cron_schedule (Optional[str]): A cron schedule string (e.g. ``"0 1 * * *"``) specifying a
+            series of times by which the `maximum_lag_minutes` constraint must be satisfied. If
+            no cron schedule is provided, then this constraint must be satisfied at all times.
+
+    .. code-block:: python
+
+        # At any point in time, this asset must incorporate all data from at least 30 minutes ago.
```

Comment:
I was showing the earlier version of the docstring to Nick, and he thought that saying "all upstream data" would make it clearer.